<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imgWindow</class>
 <widget class="QWidget" name="imgWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string>img</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="l1_p1x">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="l1_p1y">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="l1_p2y">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="l1_p2x">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="l2_p1y">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="l2_p2x">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="l2_p1x">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="l2_p2y">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l1_res">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="l2res">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>230</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sel1">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
